test/integration.html

```diff
@@ -12,8 +12,20 @@
 <head>
   <title>iron-location</title>
 
-  <script src="../../webcomponentsjs/webcomponents.js"></script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
   <script src="../../web-component-tester/browser.js"></script>
+  <script>
```

Comment:
I see. WDYT of moving this into its own file and load it here? e.g. `<script src="url-enforcer.js"></script>`